docs/manual/scala/gettingstarted/IntroGetStarted.md

```diff
@@ -67,8 +67,10 @@ It will take a bit of time to build the project and start the services. Among ot
 [info] Cassandra server running at 127.0.0.1:4000
 [info] Service locator is running at http://localhost:9008
 [info] Service gateway is running at http://localhost:9000
-[info] Service hello-impl listening for HTTP on 0:0:0:0:0:0:0:0:24266
-[info] Service hello-stream-impl listening for HTTP on 0:0:0:0:0:0:0:0:26230
+[info] Service hello-impl listening for HTTP on 127.0.0.1:24266
```

Comment:
There is also the `IntroGetStarted.md` for Java.

Reply:
There is but in the Java version the contents are organized in a slightly different way. In the Java equivalent, the change is on `GettingStartedMaven.md` _and_ `GettingStartedSbt.md`.